src/identity/apis/org.ts

```diff
@@ -101,21 +100,20 @@ export const fetchDefaultAccountDefaultOrg = async (): Promise<
   if (response.status === 500) {
     throw new ServerError(response.data.message)
   }
-  const {data} = response
+  const {data: accounts} = response
 
-  if (Array.isArray(data) && data.length) {
-    const defaultAccount = data.find(account => account.isDefault)
+  if (Array.isArray(accounts) && accounts.length) {
+    // if no default Account, reference the 0-indexed account
+    const defaultAccount =
+      accounts.find(account => account.isDefault) || accounts[0]
```

Comment:
Behavior is probably the same either way in this case, but might be better to use nullish coalescing (`??`) over or (`||`) here, since the only behavior we're trying to guard against is `.find` returning `undefined`